.travis.yml

```diff
@@ -13,7 +13,7 @@ install:
   - python setup.py install
 
 script:
-  - pytest -x --cov=./ -d --tx 3*popen//python=python3.6 --pyargs tests
```

Comment:
I would rather that the tests fail on the server asap. One can then run the tests locally for full coverage.

But I don't have a strong opinion.

Reply:
Disagree, travis and developer's environments are different and can produce unlike results, which *must* not happen, but still it is important to see whole report and all errors.